tools/helixer/.shed.yml

```diff
@@ -0,0 +1,7 @@
+categories: [Genomics Analysis]
+description: Setup and train models for de novo prediction of gene structure 
```

Comment:
I would use "Gene calling with Deep Neural Networks." instead (here and for long_description)